cmd/main.go

```diff
@@ -131,7 +136,8 @@ func run() int {
 	var npaClient *npaclient.Clientset
 	var npaInformerFactory npainformers.SharedInformerFactory
 	var nodeInformer coreinformers.NodeInformer
-	if adminNetworkPolicy || baselineAdminNetworkPolicy {
+
+	if adminNetworkPolicy || baselineAdminNetworkPolicy || clusterNetworkPolicy {
```

Comment:
do you want to check somewhere that ANP or BANP shouldn't be enabled together with CNP?

Reply:
Should we? I think it makes sense to be One of as both sounds exclusive

Reply:
I will do it now